aws/commands.yaml

```diff
@@ -16,3 +16,12 @@ commands:
     description: Know the status of your EC2 instances.
     parameters:
       - name: id
+  dynamodb_list:
+    sourcePath: dynamodb/dynamodb_list.js
+    description: List DB Tables under your account.
+    parameters:
+      - name: startTable
+        optional: true
+  rds_list:
+    sourcePath: rds/rds_list.js
+    description: List RDS instances under your account.
```

Comment:
Expand RDS for people not knowing what it means

Reply:
Updated!